HomeWork_14/HW14.py

```diff
@@ -0,0 +1,50 @@
+import re
+import json
+from datetime import datetime
+
+
+try:
+    with open('/Json_hw14', 'r') as file:
```

Comment:
`/` at the beginning of the filename tells that it is an absolute path. It means that the file will be created in the root of a filesystem which is not good. Remove that slash ('/')